apps/frontend/tests/fixtures/test-with-registered-user.ts

```
@@ -15,54 +16,55 @@ type TestWithRegisteredUser = {
-			/**
-			 * This happens after each test that uses this fixture.
-			 */
-			await cleanup(id);
-		},
-		{ scope: "test", auto: true },
-	],
-});
+export const testWithRegisteredUser =
+	testWithoutSplashScreen.extend<TestWithRegisteredUser>({
+		account: [
+			async ({}, use) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)